.github/workflows/tests.yaml

```diff
@@ -71,20 +152,14 @@ jobs:
     if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: actions/download-artifact@v2
       with:
-        python-version: '3.7'
-    - name: Install build and publish tools
-      run: |
-        pip install build twine
-    - name: Build and check
-      run: |
-        rm -rf dist/ && python -m build
-        twine check --strict dist/*
```

Comment:
Does this still https://github.com/pypa/gh-action-pypi-publish#disabling-metadata-verification